gen3/neural-networks/basic-example/utils/arguments.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+from typing import Tuple
+import os
+
+
+def initialize_argparser():
+    value = os.getenv("DEPTHAI_DEVICE_NAME_LIST")
+    value = value if value else ""
+    
+    """Initialize the argument parser for the script."""
+    parser = argparse.ArgumentParser()
+    parser.description = "General example script to run any model available in HubAI on DepthAI device. \
+        All you need is a model slug of the model and the script will download the model from HubAI and create \
+        the whole pipeline with visualizations. You also need a DepthAI device connected to your computer. \
+        Currently, only RVC2 devices are supported. If using OAK-D Lite, please set the FPS limit to 28."
+
+    parser.add_argument(
+        "-s",
+        "--model_slug",
+        help="Slug of the model copied from HubAI.",
+        required=True,
+        type=str,
+    )
+
+    parser.add_argument(
+        "-fps",
+        "--fps_limit",
+        help="FPS limit for the model runtime.",
+        required=False,
+        default=30.0,  # default DepthAI FPS value
+        type=float,
+    )
+    
+    parser.add_argument(
+        "-ip",
```

Comment:
Let's change this to:
`parser.add_argument("-d", "--device", type=str, default="", help="Optional name, DeviceId or IP of the camera")`